### PR TITLE
Slight change in Pod Disruption Budget example

### PR DESCRIPTION
**Pull request by liviux**
Made example part of Pod Disruption Budget easier to understand

### CODE REVIEW

File: content/en/docs/concepts/workloads/pods/disruptions.md [hunk 1]
```diff
@@ -157,8 +157,8 @@ Initially, the pods are laid out as follows:
 | pod-a  *available*   | pod-b *available*   | pod-c *available*  |
 | pod-x  *available*   |                     |                    |
 
-All 3 pods are part of a deployment, and they collectively have a PDB which requires
-there be at least 2 of the 3 pods to be available at all times.
+The 3 pods (`pod-a`, `pod-b`, and `pod-c`) are part of a deployment, and they collectively have a PDB which requires
+there be at least 2 of the 3 pods for each to be available at all times.
```
Comment (kundan2707) [inline content/en/docs/concepts/workloads/pods/disruptions.md:161]:
```suggestion
All 3 pods (`pod-a`, `pod-b`, and `pod-c`) are part of a deployment, and they collectively have a PDB which requires
there be at least 2 of the 3 pods to be available all times.
```

Comment (kbhawkey) [reply content/en/docs/concepts/workloads/pods/disruptions.md:161]: @liviux ,
Do you agree with the suggestion?

Comment (sftim) [reply content/en/docs/concepts/workloads/pods/disruptions.md:161]: I'd capitalize the first letter of Deployment, and I'd write PodDisruptionBudget out longhand.

Also, I prefer “that requires” over “which requires”.

@kundan2707 is your suggestion the same as skipping this PR? If not, what did I miss?

**Comment by reylejano** [comment]
Hi @liviux , I'm closing this PR since there hasn't been traction.
When you have time to address the comments below, feel free to reopen this PR with `/reopen` in a comment
- https://github.com/kubernetes/website/pull/44076/files#r1444675635
- https://github.com/kubernetes/website/pull/44076/files#r1430673043
- https://github.com/kubernetes/website/pull/44076#discussion_r1404537492

/close

**Pull request closed without merging**